### ISSUE TITLE
Support bundling outside the Inbox

### ISSUE THREAD

**Issue by menturi** [leggett/simplify#665]
Currently, when in the main email list view, clicking on a label (i.e., a tag) has no special behavior—the email opens normally, as if I clicked anywhere else on the email line.

This feature request entails having each of the labels shown in the email list view be a link. Clicking the link will append a filter to the search that includes the clicked label. This will make it easier to navigate labels and to search multiple labels simultaneously without needing to type in the search bar.

Example:

- When in the Inbox list view, clicking a label called _TestLabel_ from some email entry will search `in:inbox in:TestLabel`.
- When already viewing the search result `is:unread in:HelloWorld`, clicking _TestLabel_ will search `is:unread in:HelloWorld in:TestLabel`.

It is worth noting how Gmail already has a similar feature, though not identical. When viewing an email with the tag _TestLabel_, it shows at the top next to the subject line. Hovering over this will bring up the text "Search for all messages with label TestLabel". Clicking on this label will search `in:TestLabel`, overwriting any previous search query.

The requested feature is different in two ways. First, the proposal acts when viewing from the email list, rather than from within an email. Second, the proposal appends to the current search, rather than starting a new search.

**Comment by leggett** [comment]
Thanks for the feature suggestion. I don't think this is a good idea (will explain) but I think there is another way I might support your need. 

Why I don't think it is a good idea:

- Clicking on a label and having that modify the current search instead of going to the label does not sound intuitive or expected
- Having clicking on the label do anything other than open the thread also sounds risky (labels can easily take up most of the row if they are long or many). 

What might work instead:

- I've been working on bundling ([video demo](https://www.youtube.com/watch?v=CPTcZKcFtMA)) in the v3 beta. 
- Right now, bundling only works in the inbox. But there isn't any reason I couldn't let you turn it on in other views. 
- This would effectively group messages in the list by label _and_ link to a search that was the current view plus the label (exactly what you're asking for).

If that sounds good, I'd be happy to leave this open and rename it, "support bundles outside the Inbox"

**Comment by menturi** [comment]
I completely agree with your reasoning on why you do not think it is a good idea.

Bundling does look like a comparable feature that seems like a good alternative solution. It would be nice it bundles appear in other views as well, so I agree on your suggestion on renaming this issue.
